### PR TITLE
compatible logs for the customized partitioner

### PR DESCRIPTION
**Pull request by DavidZeLiang**
fix reference issue

### CODE REVIEW

File: decisionai_plugin/common/util/kafka_operator.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 from .constant import IS_INTERNAL, IS_MT
 import json
 from .kafka_util import RoundRobinPartitioner
-
+json.dumps()
```
Comment (minkefusiji) [inline decisionai_plugin/common/util/kafka_operator.py:11]: Remove this

File: decisionai_plugin/common/util/kafka_operator.py [hunk 2]
```diff
@@ -9,8 +9,6 @@
 import json
 from .kafka_util import RoundRobinPartitioner
 
-producer=None
-
```
Comment (minkefusiji) [inline decisionai_plugin/common/util/kafka_operator.py:13]: What are you doing here?

Comment (minkefusiji) [reply decisionai_plugin/common/util/kafka_operator.py:13]: Why remove producer?

Comment (DavidZeLiang) [reply decisionai_plugin/common/util/kafka_operator.py]: typo. fixed

**Pull request merged**